examples/tutorial/jupyter/execution/pandas_on_unidist/setup_kernel.py

```diff
@@ -0,0 +1,23 @@
+import sys
+from ipykernel import kernelspec
+
+default_make_ipkernel_cmd = kernelspec.make_ipkernel_cmd
+
+
+def new_make_ipkernel_cmd(
+    mod="ipykernel_launcher", executable=None, extra_arguments=None
+):
+    mpi_arguments = ["mpiexec", "-n", "1"]
+    arguments = default_make_ipkernel_cmd(mod, executable, extra_arguments)
+    return mpi_arguments + arguments
```

Comment:
This is not applicable because we must not change the arguments for the function that is being replaced in the library.

Reply:
I think this is applicable because we shouldn't override default values for arguments of `default_make_ipkernel_cmd`. Those might be changed in the original package at some point.

Reply:
not addressed yet